augur/filter/_run.py

```diff
@@ -80,11 +80,13 @@ def run(args):
         sequence_index_ids = set(sequence_index.index.values)
 
     # Check ids for duplicates and compare against sequence index.
-    if args.sequences:
+    if args.skip_checks:
+        print_debug(f"Skipping first pass of sequence file due to --skip-checks.")
```

Comment:
This ends up being the only new, uncovered line. Might be nice to add a test with `--skip-checks`.

Reply:
Added:

https://github.com/nextstrain/augur/blob/0d15bb650e52f87e75621f633b43212f37b8c21f/tests/functional/filter/cram/filter-duplicates-error.t#L94-L117